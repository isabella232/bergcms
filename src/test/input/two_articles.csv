1	The First Chapter	900	First Title	F	First<br>Head	<br>Frist Body<br>	Last<tab>Footer	20160319-125207-Sa / 127.0.0.1:52134 / leutloff
42	\textbf{chapter "with quotes"}	100	\section{"title"}	A1	This is the beginning of the "article".<br>This is the second line.	The main part.<br><br>""""\\""	The last line.<br>Now some nasty content öäüÖÄÜ"`'"\n\t\\\".<br>Hope everything works... ;-)	20160319-124907-Sa / 127.0.0.1:52134 / leutloff
