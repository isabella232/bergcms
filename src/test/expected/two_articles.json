Content-type: application/json; charset=utf-8
Status: 200

{
    "articles": [
        {
            "id": "1",
            "priority": "900",
            "type": "F",
            "chapter": "The First Chapter",
            "title": "First Title",
            "header": "First\nHead",
            "body": "\nFrist Body\n",
            "footer": "Last\tFooter",
            "lastChanged": "20160319-125207-Sa \/ 127.0.0.1:52134 \/ leutloff"
        },
        {
            "id": "42",
            "priority": "100",
            "type": "A1",
            "chapter": "\\textbf{chapter \"with quotes\"}",
            "title": "\\section{\"title\"}",
            "header": "This is the beginning of the \"article\".\nThis is the second line.",
            "body": "The main part.\n\n\"\"\"\"\\\\\"\"",
            "footer": "The last line.\nNow some nasty content öäüÖÄÜ\"`'\"\\n\\t\\\\\\\".\nHope everything works... ;-)",
            "lastChanged": "20160319-124907-Sa \/ 127.0.0.1:52134 \/ leutloff"
        }
    ]
}
